tests/ci_build/Dockerfile.mklml_gpu

```diff
@@ -0,0 +1,14 @@
+FROM nvidia/cuda:7.5-cudnn5-devel
+# the reason we used a gpu base container because we are going to test MKLDNN
+# operator implementation against GPU implementation
+
+COPY install/ubuntu_*.sh /install/
+
+RUN /install/ubuntu_install_core.sh
+RUN /install/ubuntu_install_python.sh
+RUN /install/ubuntu_install_scala.sh
+
+RUN wget --no-check-certificate -O /tmp/mklml.tgz https://github.com/dmlc/web-data/raw/master/mxnet/mklml-release/mklml_lnx_2017.0.2.20170209.tgz
```

Comment:
added a comment in `prepare_mkl.sh` that one should also update this dockfile if there is a new release